extensions/Worker.Extensions.Kafka/src/KafkaOutputAttribute.cs

```diff
@@ -32,38 +32,38 @@ public KafkaOutputAttribute(string brokerList, string topic)
         /// Gets or sets the Avro schema.
         /// Should be used only if a generic record should be generated
         /// </summary>
-        public string? AvroSchema { get; set; }
+        public string AvroSchema { get; set; }
```

Comment:
Why changing from nullable to non nullable? Does tooling/static analysis shows a warning about this (non nullable)prop not being initialized? I think it is best to embrace nullable type feature the language is offering.

Will customers start seeing warnings about props not being initialized after they upgrade from 3.6.0 to 3.70?

Reply:
The reason for the change is the compilation error with use of nullable attributes as discussed [here](https://github.com/Azure/azure-functions-kafka-extension/pull/362). 
Agreed that we should embrace the nullable types - something we will definitely look into going forward. But for now - In order to maintain parity with the kafka extension we are keeping the attributes non-nullable.